packages/core/src/components/event-list/event-list.tsx

```diff
@@ -79,26 +79,45 @@ export class EventList {
 
   private onMutation(mutationRecords: Array<MutationRecord>) {
     this.triggerFadeOut().then(() => {
-      if (typeof this.itemHeight === 'number') {
-        const height = convertToRemString(this.itemHeight);
-
-        mutationRecords
-          .filter((mutation) => mutation.type === 'childList')
-          .forEach((mutation) =>
-            mutation.addedNodes.forEach((item) => {
-              const itemHtml = item as HTMLElement;
-
-              this.setCustomHeight(itemHtml, height);
-            })
-          );
+      if (!Number.isNaN(Number(this.itemHeight))) {
+        const height = convertToRemString(this.itemHeight as number);
+        const eventListItems = this.findEventListItems(mutationRecords);
+        eventListItems.forEach((item) => this.setCustomHeight(item, height));
       }
 
       this.handleChevron(this.chevron);
-
       this.triggerFadeIn();
     });
   }
 
+  private findEventListItems(mutationRecords: MutationRecord[]): HTMLElement[] {
+    const eventListItems: HTMLElement[] = [];
+
+    mutationRecords.forEach((mutation) => {
+      if (mutation.type !== 'childList') {
+        return;
+      }
+
+      mutation.addedNodes.forEach((node) => {
+        if (node.nodeType !== Node.ELEMENT_NODE) {
+          return;
+        }
+
+        const element = node as HTMLElement;
+
+        if (element.tagName === 'IX-EVENT-LIST-ITEM') {
+          eventListItems.push(element);
+        }
+
+        eventListItems.push(
+          ...Array.from(element.querySelectorAll('ix-event-list-item'))
+        );
+      });
```

Comment:
What is this supposed to do? Nested list items should get their own mutation entry.